src/helpers/getBadge.ts

```diff
@@ -0,0 +1,16 @@
+import { DataKey } from '@/models/DataKey'
+import ledgers from '@/helpers/ledgers'
+
+export default async function (tokenAddress: string) {
+  for (const [ledgerName, ledger] of Object.entries(await ledgers)) {
```

Comment:
You can get rid of `{}` for each `for` :)

Reply:
Why? :)

Reply:
or, thank you, I didn't know that